06_HelloGraphicsQueue/main.cpp

```diff
@@ -47,6 +47,32 @@ class HelloGraphicsQueueApp final : public examples::MonoDeviceApplication, publ
 			// - save image to disk
 			// all without using IUtilities
 
+			auto redImage = core::make_smart_refctd_ptr< IGPUImage >;
+			auto blueImage = core::make_smart_refctd_ptr< IGPUImage >;
```

Comment:
You can't just do make_smart_refctd_ptr with nothing, what this does is call the constructor with the arguments you provided and then make it into a smart_refctd_ptr. In the case of IGPUImage, you need to create it using your logical device.

Here's an example of creating an IGPUImage from the example 62 I'm working on:
```cpp
			nbl::video::IGPUImage::SCreationParams imgInfo;
			imgInfo.format = EF_R8G8B8A8_SRGB;
			imgInfo.type = nbl::video::IGPUImage::ET_2D;
			imgInfo.extent.width = w;
			imgInfo.extent.height = h;
			imgInfo.extent.depth = 1u;
			imgInfo.mipLevels = 1u;
			imgInfo.arrayLayers = 1u;
			imgInfo.samples = nbl::asset::ICPUImage::ESCF_1_BIT;
			imgInfo.flags = nbl::asset::IImage::E_CREATE_FLAGS::ECF_NONE;
			imgInfo.usage = nbl::asset::IImage::EUF_STORAGE_BIT | nbl::asset::IImage::EUF_TRANSFER_DST_BIT;
			imgInfo.initialLayout = nbl::video::IGPUImage::EL_UNDEFINED;
			imgInfo.tiling = nbl::video::IGPUImage::ET_OPTIMAL;

			auto image = device->createImage(std::move(imgInfo));
			auto imageMemReqs = image->getMemoryReqs();
			imageMemReqs.memoryTypeBits &= device->getPhysicalDevice()->getDeviceLocalMemoryTypeBits();
			device->allocate(imageMemReqs, image.get());

```

You'd need to set your width x height (512x512 and 256x256 from what I understood) and check the usages and such